src/net.cpp

```diff
@@ -1255,9 +1255,22 @@ void CConnman::InactivityChecks()
     }
 }
 
+void CConnman::NotifyNumConnectionsChanged()
```

Comment:
In commit "Move NotifyNumConnectionsChanged logic to private method." (e817ce3090fa71c61170216090685ee142d0f1d5)

It would be nice if you combined this commit with commit "Move DisconnectNodes logic to private method" (2d5c345507208d215c0621180305a9f1d363012a), or moved this commit before it, so this block of code only moves once instead of twice inside the same PR.

Reply:
I've restructured the first commit so that the logic isn't moved the first time.